eclair-core/src/main/scala/fr/acinq/eclair/api/Service.scala

```diff
@@ -225,7 +225,9 @@ trait Service extends Logging {
                         // the amount is now given with the description
                         case JInt(amountMsat) :: JString(description) :: Nil =>
                           completeRpcFuture(req.id, (paymentHandler ? ReceivePayment(Some(MilliSatoshi(amountMsat.toLong)), description)).mapTo[PaymentRequest].map(PaymentRequest.write))
-                        case _ => reject(UnknownParamsRejection(req.id, "[description] or [amount, description]"))
+                        case JInt(amountMsat) :: JString(description) :: JInt(expirySeconds) :: Nil =>
+                          completeRpcFuture(req.id, (paymentHandler ? ReceivePayment(Some(MilliSatoshi(amountMsat.toLong)), description, Some(expirySeconds.toLong))).mapTo[PaymentRequest].map(PaymentRequest.write))
+                        case r => reject(UnknownParamsRejection(req.id, "[description] or [amount, description] or [amount, description, expiryDuration] : "+r.toString()))
```

Comment:
the more complete rejection handler is not necessarily a bad idea, but that is the kind of thing that needs to be addressed everywhere properly and not in a case-by-case basis, otherwise we end up with inconsistent code.